lib/ClientBase.js

```diff
@@ -240,6 +240,9 @@ ClientBase.prototype._getAllHttp = function(opts, callback, headers) {
     if (this.hasField(opts, 'order')) {
       args.order = opts.order;
     }
+    if (path === 'transactions') {
```

Comment:
path is `colName` for context

https://github.com/cryptotaxcalculator/coinbase-node/blob/33212ff347360574ad1729ef8d000b8af72776ef/lib/ClientBase.js#L222-L230

Reply:
the `getTransactions` method 

https://github.com/cryptotaxcalculator/coinbase-node/blob/33212ff347360574ad1729ef8d000b8af72776ef/lib/model/Account.js#L98-L106

Reply:
seems like we can change/remove this package in our BE, but having look at the coinbase oauth code, I think it's a big change and high risk, but I can work on this in the future

Reply:
@MacMacky Do we know what the next_uri contains? If it doesn't keep the expand=all then line 226 would cause us to loose this data. I'm thinking it probably includes the args in the query string though?

Reply:
We could also make this an option like the others but I think it's fine to default all imo

Reply:
> @MacMacky Do we know what the next_uri contains? If it doesn't keep the expand=all then line 226 would cause us to loose this data. I'm thinking it probably includes the args in the query string though?


@JoelBehlevanas yeah, mb, `next_uri` it does not include the `expand` query, i've updated it
<img width="866" alt="Screenshot 2023-12-06 at 4 44 49 PM" src="https://github.com/cryptotaxcalculator/coinbase-node/assets/32118627/b0a0ea76-1555-43a9-a206-07fa97db077e">